package.json

```diff
@@ -79,7 +79,7 @@
     "dayjs": "^1.8.16",
     "domready": "^1.0.8",
     "form-serialize": "^0.6.0",
-    "genome-visualizer": "git://github.com/ENCODE-DCC/valis-hpgv.git#1.3.0",
+    "genome-visualizer": "git://github.com/ENCODE-DCC/valis-hpgv.git#VALIS-14-show-error-on-invalid-loc",
```

Comment:
This is good but it looks like package-lock.json is pointing to commit with SHA:

> "version": "git://github.com/ENCODE-DCC/valis-hpgv.git#b5c9bb34f2747a15a4870b9c6955bfdd08c790a2"

While the SHA for the corresponding VALIS-4 branch is:

> 07b1a6dddc561b66a6b3ae7da0c6f68812a47352

Shouldn't these match?

Reply:
This has been updated.